### DISCUSSION TITLE
Mountain Creator

### DISCUSSION THREAD

**Discussion by shplim** [OpenRCT2/OpenRCT2#22821] [Feature requests]
The current land elevation tool simply creates a pyramid-like mound that sprawls over a large area whether you want it to or not.

I'm hoping in the future there will be a way to control the size of the base of the mountain, but still maintain a level of proportionality with the height of the land creation. Think more isoceles triangle, rather than the right-angled mounds we can currently create.

I think the best way to do this is is to type in a number for a square mountain base size and have just a single square of land to control and raise, being able to also select corners.

There should also be an option to make the mountain even on all sides or randomize the ruggedness of the slopes, perlin noise if possible.

I expect this will cause bare vertical faces on much of the mountain, some longer than others.

**Comment by X123M3-256** [comment]
There's a plugin for this https://openrct2plugins.org/plugin/R_kgDOMo-Qtw/openrct2-worldpainter
